mu-plugins/blocks/global-header-footer/blocks.php

```diff
@@ -975,6 +975,12 @@ function set_current_item_class( $menu_items ) {
 
 	foreach ( $menu_items as & $item ) {
```

Comment:
The check could probably be done outside of the loop, to reduce the calls to `get_download_url()`:

`if ( get_download_url() === $current_url ) return $menu_items;`

The only times when this inner item would be needed is if it's going to highlight the downloads url, which means it must be the `$current_url`, right?

Reply:
Yep that sounds good.

In order to get the `$current_url` working locally, I updated 

```
if ( ! $url ) {
	$url = 'https://wordpress.org/download/';
}
```
**to**

```
if ( ! $url ) {
	$host = strtolower( $_SERVER['HTTP_HOST'] );
	$url  = "https://$host/download/";
}
```

On paper, this seems like an innocuous change but can i be missing something? I don't mind backing it out if it was intentional.

Reply:
Should be right, but given the rest of the file has hard-coded URLs for menus, I'm not sure it makes much difference.

The change would mean that on Rosetta sites, if it can't find a downloads page for some reason, it'll load a 404 `/downloads` instead of reverting to using the primary page.

Reply:
No, they don't. I'll just remove it.